spaceship/lib/spaceship/tunes/application.rb

```diff
@@ -201,8 +201,8 @@ def build_trains
 
       # The numbers of all build trains that were uploaded
       # @return [Array] An array of train version numbers
-      def all_build_train_numbers
-        client.all_build_trains(app_id: self.apple_id).fetch("trains").collect do |current|
+      def all_build_train_numbers(platform: nil)
```

Comment:
Adding the new parameter will not work with accessing this from `download_dsyms`, this have to be 2 separate PRs with a dependency update. So first step would be to split the PR up to be 2 PRs 👍